sdks/python/apache_beam/runners/interactive/display/display_manager.py

```diff
@@ -47,6 +47,11 @@ def _formatter(string, pp, cycle):  # pylint: disable=unused-argument
   IPython = None
   _display_progress = print
 
+try:
```

Comment:
For consistency with other parts of beam, we can use: `from past.builtins import unicode`